File: Server/SuperBoard/src/main/java/com/narara/superboard/cardmember/interfaces/dto/log/RepresentativeStatusChangeInfo.java
package com.narara.superboard.cardmember.interfaces.dto.log;


import com.narara.superboard.card.document.CardInfo;

// CardMember 대표 상태 변경 관련 정보
public record RepresentativeStatusChangeInfo(
        Long memberId,
        String memberNickname,
        Long cardId,
        String cardName,
        boolean isRepresentative
) implements CardInfo {
}

File: Android/core/model/src/main/java/com/ssafy/model/activity/cardmember/RepresentativeStatusChangeInfo.kt
package com.ssafy.model.activity.cardmember

data class RepresentativeStatusChangeInfo(
    val memberId: Long,
    val memberNickname: String,
    val cardId: Long,
    val cardName: String,
    val isRepresentative: Boolean,
) {
    override fun toString(): String {
        return memberNickname
    }
}
